File: Javatraining/src/com/Assignments/Day3/Address.java
package com.Assignments.Day3;

public class Address {
	int Doorno;
	String street;
	int  pincode;
	String city;
	
	Address(){
		
	}
	Address(int doorno, String street, int pin, String city){
		this.Doorno=doorno;
		this.street=street;
		this.pincode=pin;
		this.city=city;
				}
	@Override
	public String toString() {
		return "Address [Doorno=" + Doorno + ", street=" + street + ", city=" + city + ", pincode=" + pincode + "]";
	}
}

File: TypeScript/TypescriptExample/Address.ts
export class Address{
    streetname:string;
    city:string;
    state:string;
constructor(street:string,city:string,state:string){
    this.streetname=street;
    this.city= city;
    this.state=state;
}
    display(){
        console.log(this.streetname);
        console.log(this.city);
        console.log(this.state);
    }
}